idaloader.cpp

```diff
@@ -253,6 +301,17 @@ void idaapi load_file(linput_t *li, ushort /*_neflags*/, const char * /*fileform
     if (file.header().Magic == MAGIC_XEX2)
       add_til("x360.til", 0);
 
+    //----------------------------------------------------------------------
+    //Let's load our xml file here, alerting the user if we don't find one
+    char new_path[500];
+    sprintf_s(new_path, 500, "%s\\idaxex.xml", idadir(LDR_SUBDIR));
+    // our xml file
+    pugi::xml_document doc;
+    pugi::xml_parse_result result = doc.load_file(new_path);
+    if (!result)
+        msg("Failed to load import/export names from XML file: %s\n", new_path);
```

Comment:
does msg(..) stop processing here, or does it continue execution. what happens if it fails, and execution continues to DoNameGen(..) below?